packages/i18n/src/next.mjs

```diff
@@ -7,17 +7,20 @@ import enNamespaces from "./next/en/index.mjs"
 import esNamespaces from "./next/es/index.mjs"
 import frNamespaces from "./next/fr/index.mjs"
 import nlNamespaces from "./next/nl/index.mjs"
+import ukNamespaces from "./next/uk/index.mjs"
 
 export const de = deNamespaces
 export const en = enNamespaces
 export const es = esNamespaces
 export const fr = frNamespaces
 export const nl = nlNamespaces
+export const uk = ukNamespaces
 
 export const languages = {
   de: "Deutsch",
   en: "English",
   es: "Español",
   fr: "Français",
-  nl: "Nederlands"
+  nl: "Nederlands",
+  uk: "undefined"
```

Comment:
I've been avoiding committing all the Uk files, but probably that needs to happen